### PR TITLE
OEC Admin View Epic, Phase 1

### PR DESCRIPTION
**Pull request by JordanGuinn**
## Background
This PR includes all the admin view work we've completed up to this point.  Given the minimal number of tickets we have outstanding in this epic, now seems like as good a time as any to ship what we've already got.

## GitHub Issue
N/A

## Validation Plan
[See below](https://github.com/ctoec/data-collection/pull/1399#issuecomment-880168964).

## Automated Testing
- [ ] All unit tests are passing.

### CODE REVIEW

**Review by vonbearshark — APPROVED** [review]
No surprises from my review

File: client/src/containers/Organizations/OrganizationsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+);
+
+const TableColumns: () => Column<OrganizationSummary>[] = () => [
+  {
+    name: 'Name',
+    sort: (row) => row.providerName,
+    cell: ({ row }) => (
+      <th scope="row">
+        {/* Deactivate links until org detail pages are built */}
+        {/* <Link to={`organization/${row.id}`}>{row.providerName}</Link> */}
```
Comment (vonbearshark) [inline client/src/containers/Organizations/OrganizationsTable.tsx:30]: Has this been rolled out?

Comment (JordanGuinn) [reply client/src/containers/Organizations/OrganizationsTable.tsx:30]: Not yet - will be coming in shortly with https://app.zenhub.com/workspaces/ece-dev-board-5ff506028d35f30012e0e937/issues/ctoec/data-collection/1280, but figured we don't need to hold up the remainder of this work any longer for that.

**Review by bamader — APPROVED** [review]
No surprises from mine, either. Didn't expect to find anything horrendously destroyed in here, but I still looked for it like I was hunting wabbit 🐇

**Comment by JordanGuinn** [comment]
# Validation Plan

## Focused Regression
### Admin List Users

### Admin List Organizations

### Admin Create Organization

### Admin Edit Users

## General Regression

### Roster View

### File Upload

### Create Record

### Edit Record

### Withdraw Record

### Delete Record

### Export Roster

**Comment by JordanGuinn** [comment]
Validation complete.  The following tickets were discovered as part of our testing efforts:

- https://github.com/ctoec/data-collection/issues/1402
- https://github.com/ctoec/data-collection/issues/1403
- https://github.com/ctoec/data-collection/issues/1404
- https://github.com/ctoec/data-collection/issues/1405

None of these issues are blockers for release though, so this is ready for merge!

**Pull request merged**